foundation.e.easy-installer.yml

```diff
@@ -0,0 +1,30 @@
+app-id: foundation.e.easy-installer
+runtime: org.freedesktop.Platform
+runtime-version: 21.08
+sdk: org.freedesktop.Sdk
+command: easy-installer
+
+finish-args:
+  - "--env=SNAP_NAME=easy-installer"
+  - "--env=SNAP_USER_COMMON=/var/tmp"
+  - "--share=network"
+  - "--share=ipc"
+  - "--socket=x11"
+  - "--socket=wayland"
+  - "--device=all"
+modules:
+  - name: easy-installer
+    buildsystem: simple
+    build-commands:
+      - mv * /app
+      - install -D --mode=0644 /app/foundation.e.easy-installer.png /app/share/icons/hicolor/scalable/apps/foundation.e.easy-installer.png
```

Comment:
a PNG is NOT a scalable icon.

Reply:
also it's not square and not any size that is standard. You need at least a 128x128 px.